concourse/scripts/kibana_set_utc.rb

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env ruby
+
+require 'net/http'
+require 'json'
+
+def ingest_to_es(document, uri)
+  raise "Document is not a hash" unless document.is_a? Hash
+  Net::HTTP.new(uri.host, uri.port).start do |http|
+    req = Net::HTTP::Put.new(uri.path, 'Content-Type' => 'application/json')
+    req.body = document.to_json
+    http.request(req)
+  end
+end
+
+def date_format_not_utc(response_json)
+end
+
+es_host = ENV.fetch("ES_HOST").delete(' ')
+es_port = ENV.fetch("ES_PORT").delete(' ')
+
+es_url = "http://#{es_host}:#{es_port}"
+date_format_url = "#{es_url}/.kibana/config/4.3.1"
+date_format_uri = URI(date_format_url)
```

Comment:
Could you change these variables to `config_url` and `config_uri`?

The existing name sounds more like a verb than a noun and doesn't describe what the endpoint is (which returns more than just `dateFormat`).